lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
To be clear, folks are not supposed to be raising Elixir exceptions. The previous implementation for `Code.LoadError` was excellent: it asserted the proper arguments were expected. The current implementation adds a series of fallbacks that doesn't seem to be necessary.